src/main/react-front-end/src/APICalls/APICalls.js

```diff
@@ -59,12 +59,12 @@ export async function checkLogin(email, accept, fail){
 	    action: 'verifyEmail',
 	    email: email,
 	}).then((response) => {
-		if(!(response.status === 200))
-			callback = fail;
-		statusHandle(response, callback);
-	}).catch((error) => {
-      statusHandle(error, fail);
-    });
+		if((response.data === true)){
+			statusHandle(response, accept);
+		}else if(response.data === false){
+			statusHandle(response, fail);
+		}
+	});
```

Comment:
What if backend throws an error due to some server issues. So, other errors have to be handled. Also. I think the error handling block is still needed.

Reply:
If there is an error, the previous method would also behave the same way.
And if the server returns false, the UI is already configured to say 'Email not found or server error.'
In this scenario, only database read can be an issue, and if that happens, the framework will bombard server logs with errors.

Reply:
Agreed that there must be a catch block regardless.
Reverting the PR
@ethirocks please handle this